### ISSUE TITLE
Synchronizing

### ISSUE THREAD

**Issue by mobile26** [Kemmey/Kemmey-TeslaWatch-Public#516]
Anything new about the options for synchronizing your watch and phone? Ref #447

**Comment by Kemmey** [comment]
Yeah - that shipped with latest release :-D check the changelog from settings
